pages/k8s/how-to-cos-lite.md

```diff
@@ -109,12 +109,12 @@ Deploy the grafana-agent:
 Juju deploy grafana-agent
 ```
 
-Relate `grafana-agent` to `k8s`, `kubernetes-control-plane` and `kubernetes-worker`:
+Relate `grafana-agent` to charmed kubernetes applications:
```

Comment:
replaces `k8s` with `kubeapi-load-balancer` as i'm sure the original author mistakenly wrote here.